whois-api/src/main/java/net/ripe/db/whois/api/rest/DomainObjectService.java

```diff
@@ -96,10 +86,8 @@ public Response create(
 
             auditlogRequest(request);
 
-            final Credentials credentials = createCredentials(updateContext, passwords);
```

Comment:
We are already creating the credentials in InternalUpdatePerformed when we do updatePerformer.createUpdate